src/emu_os_bip32.c

```diff
@@ -147,6 +168,9 @@ unsigned long sys_os_perso_derive_node_bip32(cx_curve_t curve, const uint32_t *p
       memmove(tmp + 1, tmpPrivateChain, 32);
     }
     else {
+      if (curve == CX_CURVE_Ed25519) {
+        errx(1, "os_perso_derive_node_bip32: invalid path");
```

Comment:
We are hitting this error here. However, AFAIK, this restriction applied to HDW_ED25519_SLIP10 and not HDW_NORMAL.

Reply:
Ack, thanks.